bpf/lib/nodeport.h

```diff
@@ -1267,24 +1301,30 @@ int tail_nodeport_nat_ipv4(struct __ctx_buff *ctx)
 	if (dir == NAT_DIR_EGRESS) {
 		struct remote_endpoint_info *info;
 
-		if (!revalidate_data(ctx, &data, &data_end, &ip4))
-			return DROP_INVALID;
+		if (!revalidate_data(ctx, &data, &data_end, &ip4)) {
+			ret = DROP_INVALID;
+			goto drop_err;
+		}
```

Comment:
Hey Han, looks like this one changed the v4 NAT code for the perf event error reporting, but not the corresponding v6 code (`tail_nodeport_nat_ipv4()`). Do you have a chance to follow-up on the v6 as well? Thx

Reply:
yes, sorry for the miss. I can send a quick follow up.